src/hud.cpp

```diff
@@ -489,23 +491,90 @@ void drawItemStack(video::IVideoDriver *driver,
 		const ItemStack &item,
 		const core::rect<s32> &rect,
 		const core::rect<s32> *clip,
-		IGameDef *gamedef)
+		IGameDef *gamedef,
+		bool selected,
```

Comment:
sorry didn't see this, I have prepared a commit to fix this, and some other issues, see IRC if you want to review it.